code-challenges.js

```diff
@@ -21,12 +32,30 @@ var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
 var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
-
-
+// Start by creating a function that takes in array of mixed data. 
+const gimmeTheRemainder = (array) => {
+// // We are probably going to filter first in order to only extract the numbers from the mixed data array.  
+  let onlyNumbs = array.filter(numbers => typeof numbers == "number")
+  return onlyNumbs.map(numbers => numbers %3)
```

Comment:
I think it's cool you managed this in two lines. I have two returns in my homework but looking at it we could chain the .map to the array.filter and have one line. Your code is consistently easy to follow and psuedo-coded thoroughly it makes reviewing very easy. great work!